Puppetfile

```diff
@@ -79,3 +79,8 @@ mod 'datadog/datadog_agent', '1.6.0'
 
 # Used for grabbing certificates for jenkins.io
 mod 'danzilio/letsencrypt', '1.0.0'
+
+# For managing ldap
+mod 'camptocamp/openldap', '1.14.0'
+mod  'herculesteam/augeasproviders_shellvar', '2.x'
```

Comment:
It's unclear to me what these additional augeas providers do here, are they actually being used by the openldap module?